app/controllers/idv/sessions_controller.rb

```diff
@@ -60,6 +59,17 @@ def update_review
 
     private
 
+    def submit_profile
+      if @profile.submit(profile_params)
+        redirect_to idv_sessions_finance_url
+      elsif @profile.errors.include?(:ssn)
+        flash[:error] = I18n.t('idv.errors.duplicate_ssn')
+        redirect_to idv_sessions_dupe_url
```

Comment:
Good question.

The redirect means that a page reload will not try and re-post the original form, which would be really odd since the form isn't present on the error page.